usecases/traverser/explorer.go

```diff
@@ -382,7 +382,7 @@ func (e *Explorer) Hybrid(ctx context.Context, params GetParams) ([]search.Resul
 	}
 	fused := FusionReciprocal(weights, results)
 
-	if len(fused) > hybridSearchLimit+1 {
+	if hybridSearchLimit > 1 && (len(fused) > hybridSearchLimit) { //-1 is possible?
```

Comment:
Do you mean maybe `if hybridSearchLimit > 0 && (len(fused) > hybridSearchLimit`?